### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by mloning**

### CODE REVIEW

**Comment by mloning** [comment]
FYI @Riyabelle25

File: .all-contributorsrc [hunk 1]
```diff
@@ -1099,6 +1099,48 @@
...
+        "doc"
+      ]
+    },
+    {
+      "login": "chrisholder",
+      "name": "chrisholder",
+      "avatar_url": "https://avatars.githubusercontent.com/u/4674372?v=4",
+      "profile": "https://github.com/chrisholder",
+      "contributions": [
+        "code",
```
Comment (fkiraly) [inline .all-contributorsrc:1120]: should also be design, right? for the clustering and distances modules

Comment (fkiraly) [reply .all-contributorsrc:1120]: plus examples, for clustering

Comment (mloning) [reply .all-contributorsrc:1120]: Done

**Review by fkiraly — APPROVED** [review]
👍

**Pull request merged**